homeassistant/components/voip/assist_satellite.py

```diff
@@ -171,7 +171,9 @@ def async_get_configuration(
         self,
     ) -> AssistSatelliteConfiguration:
         """Get the current satellite configuration."""
-        raise NotImplementedError
+        return AssistSatelliteConfiguration(
```

Comment:
Should we put this in the base class instead?

Reply:
Moved to websocket API